src/coreclr/src/vm/class.cpp

```diff
@@ -967,12 +967,87 @@ void ClassLoader::LoadExactParents(MethodTable *pMT)
 
     MethodTableBuilder::CopyExactParentSlots(pMT, pApproxParentMT);
 
+    // Record this type for dynamic dictionary expansion (if applicable)
```

Comment:
Do we still need this?

Reply:
As discussed, I change this to be lazily updated as well whenever we call `JIT_GenericHandleWorker`